tools/upgrade_versions.py

```diff
@@ -61,6 +61,16 @@ def _sed_yaml_descriptor(yamlfile, old_commit, new_commit):
             f.write(line)
 
 
+def _sed_rpm_spec(specfile, old_commit, new_commit):
+    lines = []
+    with file(specfile, "r") as f:
+        lines = f.readlines()
+    with file(specfile, "w") as f:
+        for line in lines:
+            line = line.replace(old_commit, new_commit)
+            f.write(line)
```

Comment:
This function is identical to the one above, you could just change its variables to have more generic names.